IPLD-Path.md

```diff
@@ -0,0 +1,20 @@
+# [WIP] IPLD Path v1
+
+An IPLD Path is a string identifier used for deep references into IPLD
+graphs.
+
+IPLD Path's are constructed following the same constraints as [URI Paths](https://tools.ietf.org/html/rfc3986#section-3.3).
+
+Similarly, the string `?` is reserved for future use as a query separator.
+
+# Path Resolution
+
+Path resolution is broken into two parts: full path resolution and block level resolution.
+
+Block level path resolutionis defined by individual codecs.
+For most common codecs each path segment is a property lookup.
```

Comment:
What do you mean by "property lookup"?

Reply:
The path segment will match a property of the decoded object. Would "attribute" be a better term than "property" here?

Reply:
I would just remove that sentence. I don't see why this really matters. For CBOR the path is a the nested property.

If you keep it, I'd go for "property".

Reply:
+1 to being a little confused by this segment and possibly just skipping it.

Reply:
Ok, I'll remove it.